nextjs-wordles/public/słownik.js

```diff
@@ -1,4 +1,4 @@
-const Dicionary = fetch('/słownik.txt')
+const Dicionary = fetch('/slowa.txt')
```

Comment:
Tutaj ten słownik się zaciągnął. Może wyjście z innego brancha niż development albo dodanie do developmentu przed PR? To nic, jak zmienisz w poprzednim PR to trzeba będzie rozwiązać jeden konflikt. 

Poza tym ekstra!